File: apps/api-dotnet/src/Planora.Application/Features/Parcels/Dtos/CreateParcel/CreateParcelResponse.cs

namespace Planora.Application.Features.Parcels.Dtos.CreateParcel
{
    public sealed record CreateParcelResponse(
        Guid ParcelId,
        string Name,
        BoundingBoxDto BoundingBox,
        BoundingBoxDto BufferedBoundingBox,
        decimal Area,
        DateTime CreatedAt);
}

File: apps/frontend/src/app/features/parcels/interfaces/parcel-new/create-parcel-response.ts
export interface CreateParcelResponse {
  parcelId: string;
  name: string;
  boundingBox: {
    minX: number;
    minY: number;
    maxX: number;
    maxY: number;
  };
  bufferedBoundingBox: {
    minX: number;
    minY: number;
    maxX: number;
    maxY: number;
  };
  area: number;
  createdAt: string;
}
